docs/intro/10-development-environment.rst

```diff
@@ -72,6 +72,18 @@ Windows:
 
 .. _yarn-setup:
 
+NPM *
+-------
+
+NPM is the package manager for JavaScript and the world’s largest software registry.
+A ``package-lock.json`` file should be synced with ``yarn.lock`` by using a package.json script:
+    ``"sync-npm": "rm -f package-lock.json && npm shrinkwrap && mv npm-shrinkwrap.json package-lock.json"``
```

Comment:
The sync-npm command can be read in package.json, I think it's more important to mention why and when to run ``yarn sync-npm``

Reply:
This isn't something that a developer should worry about. We just add it to package.json and call it in postinstall. All that the developer should do to have everything working is run `yarn`.